bin/deb_local

```diff
@@ -67,7 +68,7 @@ def build_locally(packager, pkg_name, options)
             raise RuntimeError, "Local rebuild of pkg #{pkg_name} failed -- see #{rebuild_log}"
         end
         puts "############### build #{pkg_name} #####################"
-        debian_package_dir = packager.packaging_dir(options[:debian_package_name])
+        debian_package_dir = packager.packaging_dir_i(options[:debian_package_name])
```

Comment:
as I said, where is make sense you can keep it,e.g. if pkg really means pkginfo. Otherwise remove it, like for packaging_dir.